website_sale_preset_carts/templates/suspend_cart_portal.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Coop IT Easy SCRLfs <http://coopiteasy.be>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+    <template id="suspend_cart_portal" name="Suspend Cart Portal" inherit_id="portal.portal_layout">
+        <xpath expr="//div[hasclass('o_portal_my_details')]" position="after">
+            <div class="o_portal_suspend_cart_date">
+                <h3 class="page-header">
+                  Cart Suspended <a href="/cart/suspend" class="btn btn-default btn-xs">Change</a>
+                </h3>
+                <p t-if="user_id.partner_id.suspend_cart">
+                    <div>
+                        <span>Suspended from:</span> <span t-field="user_id.partner_id.cart_suspended_from"/>
+                    </div>
+                    <div>
+                        <span>Suspended until:</span> <span t-field="user_id.partner_id.cart_suspended_date"/>
```

Comment:
cfr. previous comment : maybe rename `cart_suspended_date` in `cart_suspended_to`